res/print_makepkg_config.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/bash
+
+source "${LIBRARY:-/usr/share/makepkg}/util/config.sh";
+
+load_makepkg_config;
+
+# config entries which can be overriden with environment variables; taken from util/config.sh
+for var in PKGDEST SRCDEST SRCPKGDEST LOGDEST BUILDDIR PKGEXT SRCEXT GPGKEY PACKAGER CARCH; do
```

Comment:
The alternative to hard-coding these would be to grep `util/config.sh`,
which would be just as brittle (if not more).